client/pages/login.vue

```diff
@@ -10,6 +10,9 @@ const usernamePlaceholder = getRandomDeveloperName()
 const message = ref<Nullable<string>>(null)
 const messageSeverity = ref<'success' | 'info' | 'warn' | 'error' | undefined>('error')
 
+const usernameField = ref()
+const passwordField = ref()
```

Comment:
```suggestion
const usernameField = ref<Nullable<CompElement>>(null)
const passwordField = ref<Nullable<CompElement>>(null)
```

for type safety, check the CompElement type that I made

Reply:
I wanted to give the type but I was missing the name.. thank you tho!